docs/CHANGELOG.md

```diff
@@ -70,6 +70,7 @@
 * [Changed] Update Okta documentation and examples ([#3126](https://github.com/quiltdata/quilt/pull/3126))
 * [Added] Add Amazon Athena documentation, examples and references ([#3116](https://github.com/quiltdata/quilt/pull/3116))
 * [Fixed] Fix four-deep headers so auto-link generation works ([#3100](https://github.com/quiltdata/quilt/pull/3100))
+* [Added] Using quilt3 login and aws configure with containers ([#3099](https://github.com/quiltdata/quilt/pull/3099))
```

Comment:
I don't think that we should changelog docs additions. That's really confusing. Added is for features.

Reply:
Based on review of our company [changelog design guide](https://keepachangelog.com/en/1.0.0/), removed all docs from changelog